frontend/src/components/FormElements/FormElements.tsx

```diff
@@ -73,3 +74,32 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
     );
   }
 );
+
+type DataList = {
+  id: string;
+  name: string;
+};
```

Comment:
The library looks really promising and seems to fulfill almost all of our use cases. One thing I was wondering is should the type for the datalist be a list of objects (Datalist) with property id and name. I think that a lot of what we will be representing as options in a dropdown list will be just lists of string, often from an enum class like Accessibility in types/index.ts.

Reply:
The datalist is a requirement for how the react-select library works 

It requires a `label` and a `name` and I noticed that a lot of the things have both an `id` and `name` so it made sense group them like that